.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+name: Build
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+      fail-fast: false
```

Comment:
Maybe we should consider enabling this?

Reply:
I guess it will be better to let it continue in case something else will fail it will help more with fixing.